### PR TITLE
Fix highcharts tooltip is clipped

### PR DESCRIPTION
**Pull request by tongyu-microsoft**
This PR fixes the bug where highcharts tooltip is clipped by allowing the tooltip to render outside the chart's SVG element box. Related documentation: https://api.highcharts.com/highcharts/tooltip.outside

## Description
Before:
Tooltip is clipped:
image

After:
Tooltip is all visible:
image

## Checklist

- [x] I have added screenshots above for all UI changes.
- [ ] I have added e2e tests for all UI changes.
- [ ] Documentation was updated if it was needed.

### CODE REVIEW

**Review by vinuthakaranth — APPROVED** [review]

**Review by gaugup — APPROVED** [review]

**Review by romanlutz — APPROVED** [review]

File: .github/workflows/Ci-raiwigets-python-typescript.yml [hunk 1]
```diff
@@ -108,8 +108,7 @@ jobs:
         working-directory: ${{ matrix.packageDirectory }}
 
       # Only try to upload code cov if python tests were run
-      - if: ${{ (steps.raiwidgettests.outcome == 'success') }}
-        name: Upload code coverage results
+      - name: Upload code coverage results
```
Comment (romanlutz) [inline .github/workflows/Ci-raiwigets-python-typescript.yml:111]: This lives in CI-raiwidgets-pytest.yml now since my PR merged.

Comment (tongyu-microsoft) [reply .github/workflows/Ci-raiwigets-python-typescript.yml:111]: I am going to revert all these changes after testing. There were weird issues where the E2E screenshots are not get uploaded.

File: .github/workflows/Ci-raiwigets-python-typescript.yml [hunk 2]
```diff
@@ -143,12 +142,12 @@ jobs:
           name: codecov-umbrella
           verbose: true
 
-      - if: ${{ matrix.operatingSystem != 'macos-latest' }}
-        name: Upload e2e test screen shot
+      - name: Upload e2e test screen shot
```
[the remark this answers is not in the document]
Comment (vinuthakaranth) [reply .github/workflows/Ci-raiwigets-python-typescript.yml]: @tongyu-microsoft , please keep the changes if this helps in getting screenshots.

[the remark this answers is not in the document]
Comment (vinuthakaranth) [reply .github/workflows/Ci-raiwigets-python-typescript.yml]: but may be can be added in different PR as well

**Pull request merged**